src/django/api/models/moderation_event.py

```diff
@@ -0,0 +1,117 @@
+import uuid
+from django.db import models
+from api.models.facility.facility_claim import FacilityClaim
+
+
+class ModerationEvent(models.Model):
+    '''
+    Data that is needed for moderation queue.
+    '''
+    uuid = models.UUIDField(
+        default=uuid.uuid4,
+        editable=False,
+        unique=True,
+        db_index=True,
+        help_text=(
+            'Unique identifier to make moderation '
+            'event table more reusable across the app.'
+        )
+    )
+
+    created_at = models.DateTimeField(
+        auto_now_add=True,
+        help_text='Date when the moderation queue entry was created.'
+    )
+
+    updated_at = models.DateTimeField(
+        auto_now=True,
+        help_text='Date when the moderation queue entry was last updated.',
+        db_index=True
+    )
+
+    status_change_date = models.DateTimeField(
+        null=True,
+        help_text='Date when the moderation decision was made.'
+    )
+
+    claim = models.OneToOneField(
+        FacilityClaim,
+        on_delete=models.CASCADE,
+        related_name='moderation_event',
+        help_text='Linked claim id for this production location.'
+    )
+
+    class RequestType(models.TextChoices):
```

Comment:
I recommend to move all classes like RequestType, Status & Source in the top of Moderation Event class to have better structure.

Reply:
Done, pls check.